src/pages/experiments/[experimentId]/data-processing/configure-embedding/components/CalculationConfig.jsx

```diff
@@ -52,9 +89,11 @@ const CalculationConfig = (props) => {
         <InputNumber
```

Comment:
I just noticed that the number does not change when we click the up and down arrows just next to the "minimum distance" number - could you take a look and fix it?

Reply:
Fixed.